src/entities/Place/model.ts

```diff
@@ -286,7 +286,7 @@ export default class PlaceModel extends Model<PlaceAttributes> {
         )}
         ${conditional(
           !!options.owner,
-          SQL` AND (p.owner = ${options.owner} ${
+          SQL` AND (LOWER(p.owner) = ${options.owner} ${
```

Comment:
Do we also need to lower the `options.owner`?

Reply:
It is not needed since it is normalized at controller level :)